doc/Installation/Installation-CentOS-7-Apache.md

```diff
@@ -168,6 +193,7 @@ Edit the text which says `RANDOMSTRINGGOESHERE` and set your own community strin
 ### Cron job
 
     cp /opt/librenms/librenms.nonroot.cron /etc/cron.d/librenms
+    chmod 644 /etc/cron.d/librenms
```

Comment:
Why are you setting the mode here?  The files are already 644

Reply:
When using a shell with an umask or automating stuff, might happen that those files have different mods set. It's just an idiot proof statement.